### DISCUSSION TITLE
Vscode extension error: Message header must separate key and value using :

### DISCUSSION THREAD

**Discussion by NagarajKandoor** [microsoft/vscode-discussions#691] [Extension Development QnA]
My extension was working previously why header formulation is causing issue now?

https://stackoverflow.com/questions/76652259/vscode-extension-error-message-header-must-separate-key-and-value-using

Any input and suggestion to fix this will be appreciated.

**Comment by NagarajKandoor** [comment]
Any suggestion ? what is going wrong in header formulation ?
https://stackoverflow.com/questions/76652259/vscode-extension-error-message-header-must-separate-key-and-value-using

**Reply by r3m0t** [reply]
1. Set a breakpoint where the exception is raised and look at the local variables.
2. Probably your LSP subprocess is communicating using stdout to the VS Code extension. If there is a `print()` or similar in your code then this will be read by `messageBuffer.js` as a message. But it is not correctly formatted to be a message hence the error.

**Reply by NagarajKandoor** [reply]
@r3m0t Thank you!
